benchmark/projection/include/create_data.hpp

```diff
@@ -0,0 +1,168 @@
+#include <nektar_interface/projection/device_data.hpp>
+#include <random>
+#include <sycl/sycl.hpp>
+#include <utility>
+#include <vector>
+
+#define SEED 32423
+
```

Comment:
Some comments for each of functions in this file that describe briefly what they do would be good.